components/MapView.js

```diff
@@ -399,6 +399,10 @@ class MapView extends React.Component {
     this._runCommand('fitToSuppliedMarkers', [markers, animated]);
   }
 
+  fitToCoordinates(coordinates, edgePadding, animated) {
```

Comment:
@spikebrehm I'll update it to use the default arguments.  

As for additional documentation, I included an EdgePadding type to the [MapView docs](https://github.com/airbnb/react-native-maps/pull/545/files#diff-49d288d573c50e6e6bd32d153a3d5ca1R96)